### PR TITLE
updated german translation

### PR DESCRIPTION
**Pull request by uDEV2019**

### CODE REVIEW

File: app/src/main/res/values-de/strings.xml [hunk 1]
```diff
@@ -44,6 +44,14 @@
     <string name="dark">Dunkel</string>
     <string name="x_deleted">%1$s gelöscht</string>
     <string name="undo">Abbrechen</string>
+	<string name ="normal">Normal</string>
```
Comment (lorenzovngl) [inline app/src/main/res/values-de/strings.xml:47]: Please remove this additional space here to align this line with the others.

File: app/src/main/res/values-de/strings.xml [hunk 2]
```diff
@@ -44,6 +44,14 @@
...
+    <string name ="bold">Fett</string>
+    <string name ="extra_bold">Extrafett</string>
+    <string name ="ui_settings">Benutzeroberfläche</string>
+    <string name ="top_bar_font_weight">Schriftstärke in der oberen Leiste</string>
+    <string name ="behaviour">Verhalten</string>
+    <string name ="appearance">Aussehen</string>
+
```
Comment (lorenzovngl) [inline app/src/main/res/values-de/strings.xml:54]: Please also remove this empty line, leaving only one empty line between the two lists.

**Review by lorenzovngl — CHANGES_REQUESTED** [review]

File: app/src/main/res/values-de/strings.xml (hunk 1 above)
Comment (uDEV2019) [reply app/src/main/res/values-de/strings.xml]: fixed this, but isn't it wrong that all new strings look like 

> <string name ="

i think the space between name and = is to much, so i've changed them now to 

> <string name="

File: app/src/main/res/values-de/strings.xml (hunk 2 above)
Comment (uDEV2019) [reply app/src/main/res/values-de/strings.xml]: fixed.
The second empty line is also present in strings.xml for english language

**Review by lorenzovngl — APPROVED** [review]

**Pull request merged**